components/ScheduleSection.js

```diff
@@ -0,0 +1,242 @@
+    sessionFirstSpeaker: 'Thai Pangsakulyanont (Task World)',
+    sessionSecondTopic: 'Behavior Driven Development in React Native',
+    sessionSecondAvatar: 'https://avatars1.githubusercontent.com/u/19698625?v=3&s=400',
+    sessionSecondSpeaker: 'Tanakorn numrubporn (CTO at Planforfit)'
+  },
+  {
+    id: 4,
+    type: 'BREAK',
+    time: 'Lunch Break'
+  },
+  {
+    id: 5,
+    type: 'FLASH',
+    time: '13:00 - 14:10',
+    sessionFirstTopic: 'Lightning Talk'
+  },
+  {
+    id: 6,
+    type: 'MAIN',
+    time: '14:15 - 15:00',
+    sessionFirstTopic: 'Switching from Mobile to React Web Developer',
+    sessionFirstAvatar: 'http://upic.me/i/5h/18489682_10211021272026019_3559519120929813202_o.jpg',
+    sessionFirstSpeaker: 'Chinnawat Panmonkol (Wongnai)',
+    sessionSecondTopic: 'Styled Components: Change your CSS styling mindset',
+    sessionSecondAvatar: 'http://upic.me/i/6u/18556898_1436098509744650_6799324924419962365_o.jpg',
+    sessionSecondSpeaker: 'Yanin Tuamsuk (Front-end Engineer at Jitta)'
+  },
+  {
+    id: 7,
+    type: 'BREAK',
+    time: 'Break'
+  },
+  {
+    id: 8,
+    type: 'MAIN',
+    time: '15:15 - 16:00',
+    sessionFirstTopic: 'Angualat to React without breakingthings',
+    sessionFirstAvatar: 'https://avatars3.githubusercontent.com/u/155392?v=3&s=400',
+    sessionFirstSpeaker: 'Jirat Kijlerdpornpailoj (SPRINT3r / Gucode)',
+    sessionSecondTopic: 'Testabled React: Patterns and comparison',
+    sessionSecondAvatar: 'http://upic.me/t/b3/15965740_10154900014726797_4415870819464215979_n.jpg',
+    sessionSecondSpeaker: 'Chakrit Likitkhajorn (Lead Developer at Taskworld)'
+  },
+  {
+    id: 9,
+    type: 'MAIN',
+    time: '16:00 - 16:45',
+    sessionFirstTopic: 'Workshop with Next.js',
+    sessionFirstAvatar: 'http://upic.me/t/3j/18485743_10213102449583027_7199744295355607870_n.jpg',
+    sessionFirstSpeaker: 'Nati Namvong (CTO at Sellsuki)',
+    sessionSecondTopic: 'Lessons Learned from using Next.js in Production',
+    sessionSecondAvatar: 'http://upic.me/i/pb/18589022_10209316039524510_1628867117219649509_o.jpg',
+    sessionSecondSpeaker: 'Panjamapong (PanJ) Sermsawatsri (CTO at TakeMeTour)'
+  }
+
+]
+
+export default function ScheduleSection () {
+  return (
+    <div className='schedule'>
+      <SectionTitle>Schedule</SectionTitle>
+      <div className='container'>
+        {
+          sessions.map(session => <Session key={session.id} {...session} />)
+        }
+      </div>
+      <style jsx>{`
+        .schedule {
+          text-align: center;
+        }
+        .container {
+          max-width: 1060px;
+          width: 100%;
+          margin: 0 auto;
+          text-align: left;
+        }
+        @media (max-width: 768px){
+          .container {
+            max-width: 750px;
+            text-align: center;
+          }
+        }
+      `}</style>
+    </div>
+  )
+}
+
+const Session = (props) => {
```

Comment:
I think this component is too long. Please break this component into smaller pieces.

By composing React elements (first comment) this code will break down into smaller components naturally.